cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -175,6 +175,11 @@ String getAncillaryConfigMapKeyLogConfig() {
         return "log4j2.properties";
     }
 
+    @Override
+    public String getGcLoggingOptions() {
```

Comment:
Does this overridden method serve any purpose? It is identical to the overridden one, isn't it?